src/proto/api.proto

```diff
@@ -52,6 +52,8 @@ message ParseQuery {
 message ArtifactListQuery {
   // the task from which to list the artifacts
   imandrax.task.TaskID task_id = 1;
+
+  imandrax.session.Session session = 2;
```

Comment:
I think pretty much all API requests will want the session ID at some point. Is there a way to make that a "base class" so we don't forget it?